packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -1692,6 +1692,42 @@ const tests = {
         },
       ],
     },
+    {
+      code: normalizeIndent`
+        function MyComponent() {
+          useEffect()
+          useLayoutEffect()
+          useCallback()
+          useMemo()
+        }
+      `,
+      errors: [
+        {
+          message:
+            'React Hook useEffect will crash when called with no arguments. ' +
```

Comment:
How about we explain the requirement instead of the behavior? Something like:

> React Hook useEffect requires an effect callback. Did you forget to pass a callback to the hook?

Reply:
Yes, that's much better. Thanks!